### PR TITLE
drop tables listed in #1830

### PR DESCRIPTION
**Pull request by neilisfragile**
#1830

### CODE REVIEW

**Comment by neilisfragile** [comment]
CI will fix itself on https://github.com/matrix-org/sytest/pull/598 getting merged

**Comment by richvdh** [comment]
> move to delta 55

I'd prefer it if we didn't bump the schema version number for every single change.

**Comment by neilisfragile** [comment]
Have not covered ```room_names``` and ```topic```  as referenced in https://github.com/matrix-org/synapse/issues/1830#issuecomment-479075041 because I want to see https://github.com/matrix-org/synapse/pull/4338 land first

File: synapse/storage/schema/delta/13/v13.sql [hunk 1]
```diff
@@ -13,19 +13,7 @@
...
-);
-
-CREATE TABLE IF NOT EXISTS application_services_regex(
-    id INTEGER PRIMARY KEY AUTOINCREMENT,
-    as_id BIGINT UNSIGNED NOT NULL,
-    namespace INTEGER,  /* enum[room_id|room_alias|user_id] */
-    regex TEXT,
-    FOREIGN KEY(as_id) REFERENCES application_services(id)
-);
+/* We used to create a tables called application_services and
```
Comment (richvdh) [inline synapse/storage/schema/delta/13/v13.sql]:
```suggestion
/* We used to create tables called application_services and
```

Comment (richvdh) [reply synapse/storage/schema/delta/13/v13.sql]: likewise in a number of other places

**Comment by richvdh** [comment]
btw please remember to squash-merge this. we definitely don't want all 24 commits in the repo history.

**Comment by richvdh** [comment]
(as per #synapse-dev, we could kill `full_schemas/11` altogether)

**Pull request merged**